build.yaml

```diff
@@ -92,7 +92,7 @@ build:
 
       # Download and uncompress saxon
       mkdir saxon
-      curl -L -o saxon/saxon9he.jar http://central.maven.org/maven2/net/sf/saxon/Saxon-HE/9.8.0-12/Saxon-HE-9.8.0-12.jar
+      curl -L -o saxon/saxon9he.jar https://repo1.maven.org/maven2/net/sf/saxon/Saxon-HE/9.8.0-12/Saxon-HE-9.8.0-12.jar
```

Comment:
In the future we should add saxon to the C# driver Jenkins image.

Reply:
good idea, https://datastax-oss.atlassian.net/browse/CSHARP-848